module/account/src/Resources/translations/log.en.yaml

```diff
@@ -6,7 +6,7 @@
 "Ergonode\\Account\\Domain\\Event\\UserPasswordChangedEvent": User password changed
 "Ergonode\\Account\\Domain\\Event\\UserRoleChangedEvent": User role changed from "%from%" to "%to%"
 "Ergonode\\Account\\Domain\\Event\\UserActivatedEvent": User activated
-"Ergonode\\Account\\Domain\\Event\\UserDisabledEvent": User disabled
+"Ergonode\\Account\\Domain\\Event\\UserDeactivatedEvent": User disabled
```

Comment:
incorec namespace you use Ergonode\Account\Domain\Event\User\UserDeactivatedEvent in prevous files.